crates/gem_chain_rpc/src/lib.rs

```diff
@@ -55,3 +55,24 @@ where
         (**self).get_transactions(block_number).await
     }
 }
+
+#[async_trait]
+pub trait ChainNFTProvider: Send + Sync {
+    async fn get_collectibles(
```

Comment:
let's add get collections method here too

Reply:
collections would be the first endpoint to use and then request collectibles

Reply:
Okay, I'll add it to the trait. but for TON, in ton API can't find the endpoint to get collections that a user owns. we have 2 solutions:
1. keep it unimplemented for TON.
2. Get all user NFTs search for their collection and return them.